### PR TITLE
docs(notification): adds A11y notification docs

### PR DESCRIPTION
**Pull request by aledavila**
Reference #491 

Adds a11y notification docs 

Make sure content matches: https://pages.github.ibm.com/IBMa/Carbon_Integration/Doc/Notification-doc.html

### CODE REVIEW

**Comment by snidersd** [comment]
Please fix minor typo in the Accessibility Considerations section item number 1. The word "notifiction" is misspelled and needs to be updated to "notification".  Thx

File: src/pages/components/notification/accessibility.mdx [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+            <strong>DAP Test</strong>
+            <br />- Violations
+          </StructuredListCell>
+        </StructuredListRow>
+      </StructuredListBody>
+    </StructuredListWrapper>
+  </Column>
+</Row>
+
+### macOS Screen reader tests
```
Comment (jillianhowarth) [inline src/pages/components/notification/accessibility.mdx]:
```suggestion
### macOS screen reader tests
```

File: src/pages/components/notification/accessibility.mdx [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+The close button contains an ARIA label of `aria-label="close"`, and the icon has `aria-hidden="true"` so it is ignored by assistive technologies. The `Tab` key is used to move focus to the close button within the notification and the `Space` key or `Enter` key activate the button to close the notification.
+
+## Accessibility considerations
+
+1. Avoid using a toast notifiction for critical alerts or long messages because they are timed and will disappear automatically making it difficult for people with various disabilities to get the entire message. An alert that disappears too quickly can lead to failure of the optional [WCAG 2.0 success criterion 2.2.3 (AAA)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-no-exceptions.html).
+2. Provide a means to make nonessential alerts optional to enhance usability for people with visual and cognitive disabilities. which also make meeting the requirements of the optional [WCAG 2.0 success criterion 2.2.4 (AAA)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-postponed.html) more difficult.
```
Comment (connor-leech) [inline src/pages/components/notification/accessibility.mdx]:
```suggestion
2. Provide a means to make nonessential alerts optional to enhance usability for people with visual and cognitive disabilities. which also make meeting the requirements of the optional [WCAG 2.0 success criterion 2.2.4 (AAA)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-postponed.html) more difficult.
```

This isn't a complete sentence, though I'm not sure what it's trying to say.

Comment (aledavila) [reply src/pages/components/notification/accessibility.mdx]: @snidersd do you have any thoughts on how we can change this sentence?

**Comment by vpicone** [comment]
@connor-leech are we good here?

File: src/pages/components/notification/accessibility.mdx (hunk 2 above)
Comment (snidersd) [reply src/pages/components/notification/accessibility.mdx]: @aledavila Here is a rewrite:
Provide a means to turn off nonessential alerts to enhance usability for people with visual and cognitive disabilities. Additional information is available in the [WCAG 2.0 success criterion 2.2.4 (AAA)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-postponed.html)success criterion. Note: Providing this functionality is optional.

Comment (connor-leech) [inline src/pages/components/notification/accessibility.mdx]:
```suggestion
2. Provide a means to turn off nonessential alerts to enhance usability for people with visual and cognitive disabilities. Additional information is available in the WCAG 2.0 success criterion 2.2.4 (AAA)success criterion. Note: Providing this functionality is optional.
```

Comment (connor-leech) [reply src/pages/components/notification/accessibility.mdx]: @aledavila just this, ready to rock otherwise

**Comment by snidersd** [comment]
Is it too late to change the DAP Test results to No violations? I just noticed that the status has changed.

**Pull request merged**